sparse_strips/vello_hybrid/src/gpu.rs

```diff
@@ -0,0 +1,600 @@
+                            min_binding_size: None,
+                        },
+                        count: None,
+                    },
+                ],
+            });
+        let pipeline_layout = device.create_pipeline_layout(&wgpu::PipelineLayoutDescriptor {
+            label: None,
+            bind_group_layouts: &[&render_bind_group_layout],
+            push_constant_ranges: &[],
+        });
+        let render_pipeline = device.create_render_pipeline(&wgpu::RenderPipelineDescriptor {
+            label: None,
+            layout: Some(&pipeline_layout),
+            vertex: wgpu::VertexState {
+                module: &render_shader,
+                entry_point: Some("vs_main"),
+                buffers: &[wgpu::VertexBufferLayout {
+                    array_stride: std::mem::size_of::<GpuStrip>() as u64,
+                    step_mode: wgpu::VertexStepMode::Instance,
+                    attributes: &GpuStrip::vertex_attributes(),
+                }],
+                compilation_options: PipelineCompilationOptions::default(),
+            },
+            fragment: Some(wgpu::FragmentState {
+                module: &render_shader,
+                entry_point: Some("fs_main"),
+                targets: &[Some(ColorTargetState {
+                    format,
+                    blend: Some(BlendState::PREMULTIPLIED_ALPHA_BLENDING),
+                    write_mask: ColorWrites::ALL,
+                })],
+                compilation_options: PipelineCompilationOptions::default(),
+            }),
+            primitive: wgpu::PrimitiveState {
+                topology: wgpu::PrimitiveTopology::TriangleStrip,
+                ..Default::default()
+            },
+            depth_stencil: None,
+            multisample: wgpu::MultisampleState::default(),
+            multiview: None,
+            cache: None,
+        });
+
+        let config = Config {
+            width,
+            height,
+            strip_height: 4,
+        };
+        let config_buf = device.create_buffer_init(&wgpu::util::BufferInitDescriptor {
+            label: Some("Config Buffer"),
+            contents: bytemuck::bytes_of(&config),
+            usage: wgpu::BufferUsages::UNIFORM,
+        });
+
+        // Create an initial buffer for strip instances
+        // Start with a small size, it will be resized as needed in prepare
+        let initial_strips_capacity = 64; // Arbitrary small initial capacity
+        let strips_buffer = device.create_buffer(&wgpu::BufferDescriptor {
+            label: Some("Strips Buffer"),
+            size: std::mem::size_of::<GpuStrip>() as u64 * initial_strips_capacity,
+            usage: wgpu::BufferUsages::VERTEX | wgpu::BufferUsages::COPY_DST,
+            mapped_at_creation: false,
+        });
+
+        // Create initial texture for alpha values
+        // It will be recreated if needed in prepare
+        let initial_alpha_texture_width = 64;
+        let initial_alpha_texture_height = 64;
```

Comment:
Should the initial alpha texture dimensions correspond with the number of initial sparse strips? I.e. 64 strips is (64 * 4=) 256 alpha values